apps/scan/backend/package.json

```diff
@@ -42,7 +42,6 @@
     "@votingworks/ballot-interpreter": "workspace:*",
     "@votingworks/basics": "workspace:*",
     "@votingworks/bmd-ballot-fixtures": "workspace:*",
-    "@votingworks/converter-nh-accuvote": "workspace:*",
```

Comment:
Did the VxScan backend not actually depend on anything from the converter module?

Reply:
I believe it used to offer the conversion in the app before the backends diverged, but the feature was removed. We just missed the dependency.